src/protobuf/client.proto

```diff
@@ -0,0 +1,37 @@
+syntax = "proto3";
+
+package opaque.protos;
+
+// The Client <-> Enclave service
+service ClientToEnclave {
+  // Client sends AttestationStatus to the Enclave, receives
+  // an evidence report for each enclave
+  rpc GetRemoteEvidence(AttestationStatus) returns (Evidences) {}
```

Comment:
Here we are assuming that the head node has collected all the evidence reports for the other workers right? Is this currently implemented?

Reply:
It is not. I'm not entirely sure how the client processes reports currently, because it communicates with the orchestrator but receives only a single report from `rpc_get_remote_report_with_pubkey_and_nonce`. Are all reports the same, and does the orchestrator just pick any of them to send back to the client?

Reply:
Hm I'm not sure how it was used to be done, but at least in https://github.com/opaque-systems/opaque-client/pull/101 I implemented it following your `.proto` file and the client will receive all of the reports and verify each one.